File: python_scripts/valid.py
from nltk.corpus import words

# make it a set to go faster
setofwords = set(words.words())

word = input("what word do you want me to check? ")
while (word not in setofwords):
	word = input("that wasn't a valid word ... try again ")
print("yep, that's a word")

File: r_scripts/valid.R
# install.packages("hunspell")
library("hunspell")
word <- readline("what word would  you like me to check? ")
correct <- hunspell_check(word)

while (!correct) {
  word <- readline("that wasn't a word... try again: ")
  correct <- hunspell_check(word)
}

print("yep, that's a word!")
